cfgov/apache/conf.d/alias.conf

```diff
@@ -10,7 +10,5 @@ Alias /lln8595c61g9qnvuwvtlcwo1k6kem8.html ${STATIC_PATH}/lln8595c61g9qnvuwvtlcw
 # Akamai Sureroute
 Alias /akamai/sureroute-test-object.html ${STATIC_PATH}/akamai-sureroute-test-object.html
 Alias /utilities/pages/akamai-sureroute-test-object.htm ${STATIC_PATH}/akamai-sureroute-test-object.html
-# Main static alias
-Alias /static/ ${STATIC_PATH}/
 # Security.txt for CVE disclosure
 Alias /security.txt ${STATIC_PATH}/security.txt
```

Comment:
Even after this change, won't all of the remaining `Alias /filename ${STATIC_PATH/filename` in this file still alias directly to the filesystem instead of going through WhiteNoise? Do we need to change these to redirects, or maybe Apache rewrites to avoid the extra HTTP request?

Reply:
Oops, your comment and mine landed at the same time. That makes sense.